# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: layout/generic/TextDrawTarget.h
# Commit: c09a51622e98
# Full Hash: c09a51622e988e81077c75e3aaa5a49df66284b3
# Author: Doug Thayer <dothayer@mozilla.com>
# Date: 2019-03-22 16:44:08
# Regressor Bug: 1441308
# File Overlap Count: 2
# Description:
#   Bug 1441308 - Core renderroot splitting changes r=kats,sotaro
#   
#   This is a large patch that contains all of the core changes for
#   renderroot splitting.
#   
# ==============================================================================

diff -r 737807563dd5 -r c09a51622e98 layout/generic/TextDrawTarget.h
--- a/layout/generic/TextDrawTarget.h	Thu Mar 21 23:14:26 2019 +0000
+++ b/layout/generic/TextDrawTarget.h	Fri Mar 22 03:15:14 2019 +0000
@@ -123,6 +123,8 @@
     return true;
   }
 
+  wr::RenderRoot GetRenderRoot() { return mResources->GetRenderRoot(); }
+
   wr::FontInstanceFlags GetWRGlyphFlags() const { return mWRGlyphFlags; }
   void SetWRGlyphFlags(wr::FontInstanceFlags aFlags) { mWRGlyphFlags = aFlags; }
 